recipe/meta.yaml

```diff
@@ -28,15 +28,17 @@ requirements:
     - pip
     - hatchling >=1.20.0
     - hatch-vcs
+    - hatch-fancy-pypi-readme
   run:
     - python
     - click >=8.0.0
     - mypy_extensions >=0.4.3
     - packaging >=22.0
     - pathspec >=0.9.0
     - platformdirs >=2
-    - tomli >=1.1.0             # [py<311]
+    - tomli >=1.1.0
     - typing_extensions >=4.0.1 # [py<311]
+    - typing-extensions >=4.0.1
```

Comment:
`# [py<311] `is still relevant, see https://github.com/psf/black/blob/24.10.0/pyproject.toml#L72-L73

Reply:
Thanks for spotting this! I'll go add the exception back to `toml`.